### PR TITLE
build login page with semantic ui

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #79

### CODE REVIEW

File: message-me/app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -1 +1,38 @@
-<h1>this is new.html.erb for sessions</h1>
+<h1 class="ui header center aligned">Welcome to MessageMe - a complete Chat App</h1>
+<h4 class="ui center aligned medium icon header">
+  <i class="orange circular power off icon"></i>
+  Log in to continue
```
Comment (tlkiong) [inline message-me/app/views/sessions/new.html.erb:4]: Same as previous PR comment about orphaned text

File: message-me/app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -1 +1,38 @@
...
+</h4>
+
+<div class="ui placeholder segment">
+  <div class="ui two column very relaxed stackable grid">
+    <div class="column">
+      <div class="ui form">
```
Comment (tlkiong) [inline message-me/app/views/sessions/new.html.erb:10]: Why is this not built using rails form tag?

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: message-me/app/views/sessions/new.html.erb (hunk 2 above)
Comment (LeeRenJie) [reply message-me/app/views/sessions/new.html.erb:10]: Used Semantic UI's code 😅 would it be better to use rails form tag compared to this?

Comment (tlkiong) [reply message-me/app/views/sessions/new.html.erb]: definitely. 
Please tell me why?

**Review by tlkiong — APPROVED** [review]

**Pull request merged**